app/components/Sidebar/Main.js

```diff
@@ -123,23 +123,25 @@ function MainSidebar() {
                   documents.active ? documents.active.template : undefined
                 }
               />
-              <SidebarLink
-                to="/drafts"
-                icon={<EditIcon color="currentColor" />}
-                label={
-                  <Drafts align="center">
-                    {t("Drafts")}
-                    <Bubble count={documents.totalDrafts} />
-                  </Drafts>
-                }
-                active={
-                  documents.active
-                    ? !documents.active.publishedAt &&
-                      !documents.active.isDeleted &&
-                      !documents.active.isTemplate
-                    : undefined
-                }
-              />
+              {!user.isViewer && (
```

Comment:
In order to use a policy here we might have to tackle this one first: https://github.com/outline/outline/issues/1158

So that we can have `can.createDocument` or something similar on the team policy 🤔